brew/brew.go

```diff
@@ -57,13 +57,13 @@ func (c *HomebrewClient) LatestTag(ctx context.Context) (*version.Version, error
 	out, err := pipeline.Output(
 		[]string{c.cmdPath, "info", c.getFullName()},
 		[]string{"head", "-1"},
-		[]string{"awk", "{ print $3 }"},
+		[]string{"grep", "-o", "-E", "([0-9]+\.){1}[0-9]+(\.[0-9]+)?"},
```

Comment:
regexp should be surrounded with backquote to escape backslashes🙏 

```suggestion
		[]string{"grep", "-o", "-E", `([0-9]+\.){1}[0-9]+(\.[0-9]+)?`},
```

Reply:
merged